docs/docs/client/push.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Push notifications
+slug: client/push
+---
+
+Push server will allow to applications notify the user on incoming requests for previously established connections.  
+Push notifications are being implemented for the client, when the device is locked. Though when the user application is open and session is active, all incoming requests are added to the List of PendingRequests. Once a client opens an application on its devices, he will have the option to decide on the transaction (approve or reject it).
```

Comment:
its more about calling defined endpoint with POST call you should also show here example + maybe link to it